api/apps/api/src/modules/access-control/projects-acl/project-access-control.ts

```diff
@@ -0,0 +1,21 @@
+export type Allowed = true;
+export type Denied = false;
+export type Permit = Allowed | Denied;
```

Comment:
I imagine we'll end up using these types in other `<ResourceType>AccessControl` services: maybe worth moving them to their own file and importing them here?